src/man/firejail-profile.txt

```diff
@@ -479,6 +479,9 @@ Allow tools such as strace and gdb inside the sandbox by whitelisting system cal
 .TP
 \fBapparmor
 Enable AppArmor confinement.
+.TP
+\fBapparmor profile
+Enable AppArmor confinement with a custom profile.
```

Comment:
Maybe elaborate some more conditions.
- Must the profile already be loaded by the kernel?
- is `profile` a name or a path?

Reply:
@rusty-snake [on Jul 26](https://github.com/netblue30/firejail/pull/5274#discussion_r929979529):

> Maybe elaborate some more conditions.
> 
> * Must the profile already be loaded by the kernel?

Yes; added to the man pages.

> * is `profile` a name or a path?

Renamed `profile` to `profile_name`.